ddl/ddl_worker.go

```diff
@@ -62,21 +62,24 @@ type worker struct {
 	quitCh   chan struct{}
 	wg       sync.WaitGroup
 
-	reorgCtx        *reorgCtx // reorgCtx is used for reorganization.
+	sessPool        *sessionPool // sessPool is used to new sessions to execute SQL in ddl package.
```

Comment:
this change seems not to be related to this PR.

Reply:
why not?

Reply:
@zhexuany Here often need to use ` sessCtx `.